onvm/onvm_mgr/onvm_stats.h

```diff
@@ -57,6 +57,51 @@
 #define ONVM_STR_STATS_STDERR "stderr"
 #define ONVM_STR_STATS_WEB "web"
 
+extern const char *NF_MSG[3];
+#define ONVM_STATS_MSG \
+        "\nNF TAG         IID / SID / CORE    rx_pps  /  tx_pps        rx_drop  /  tx_drop"\
+        "         out   /    tonf     /   drop\n"\
+        "--------------------------------------------------------------------------------------------------------------------\n"
+#define ONVM_STATS_ADV_MSG \
+        "\nNF TAG         IID / SID / CORE    rx_pps  /  tx_pps        rx_drop  /  tx_drop"\
+        "           out   /    tonf     /   drop\n"\
+        "               PNT / S|W / CHLD  drop_pps  /  drop_pps      rx_drop  /  tx_drop"\
+        "           next  /    buf      /   ret\n"\
+        "----------------------------------------------------------------------------------------------------------------------\n"
+#define ONVM_STATS_SHARED_CPU_MSG \
```

Comment:
How about we do it like this:
```
#define ONVM_STATS_SHARED_CPU_MSG "\n"\
        "NF TAG         IID / SID / CORE    rx_pps  /  tx_pps        rx_drop  /  tx_drop           out   /    tonf     /   drop\n"\
        "               PNT / S|W / CHLD  drop_pps  /  drop_pps      rx_drop  /  tx_drop           next  /    buf      /   ret\n"\
        "                                  wakeups  /  wakeup_rt\n"\
        "----------------------------------------------------------------------------------------------------------------------\n"
```

Even though the linter complains its a lot easier to read.

Reply:
(applies to other defs)

Reply:
Sounds good just pushed that change